packages/runtime-node/src/types.ts

```diff
@@ -148,7 +150,7 @@ export interface IExternalDefinition {
     packageName: string;
     /**
      * the directory where the built source code is located at
-     * @default dist
+     * @defaultValue dist
```

Comment:
Not sure this is an improvement

Reply:
It's the tsdocs spec, thus reduces warning during docs generations